website/docs/configuration/recommended-rbac-configuration.mdx

```diff
@@ -178,3 +180,142 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ---
 ```
+
+
+## Weave GitOps Enterprise RBAC considerations  <TierLabel tiers="enterprise" />
+
+Weave GitOps Enterprise extends Weave GitOps permissions by adding more roles. These roles may need to be extended further in order to support certain use cases. Some of the most common use cases are described below.
+
+### Viewing Flagger-generated resources
+
+Weave GitOps Enterprise integrates with Flagger in order to provide a view on progressive delivery deployments. Part of that view, is the ability to view all the resources that Flagger manages during its operation. The default ClusterRole `gitops-canaries-reader`, used for viewing canary details, includes the minimum permissions necessary for a user to be able to view canary object details, metric template object details and canary related events. When a canary relies on a service mesh such as Linkerd or Istio for the rollout, then this ClusterRole needs to be extended so that it can read the additional service mesh resources being generated by Flagger. Note that this functionality relies on each canary resource explicitly specifying the provider that it uses. The following table provides a list of all the custom resources that Flagger generates grouped by provider:
```

Comment:
nit: heavy repetition of the word view. Suggest replacing "Part of that view, is" with "This includes"

Reply:
", used for viewing canary details" is redundant as explained later.

Reply:
> When a canary relies on a service mesh such as Linkerd or Istio for the rollout,

Suggestion: When Flagger is configured to integrate with a service mesh such as Linkerd or Istio,

Reply:
> Note that this functionality relies on each canary resource explicitly specifying the provider that it uses.

Note - currently in order to display service mesh or ingress related resources, we require `spec.provider` to be set in each canary resource.

Reply:
This is a fairly large block of text, suggest new lines before:
-  "When a canary relies..." 
- "The following table"

Reply:
Updated 👍